### DISCUSSION TITLE
Vue Support

### DISCUSSION THREAD

**Discussion by sergiohdljr** [biomejs/biome#1678] [Ideas]
There's been any news or updates on Vue support ?

**Comment by ematipico** [comment]
I suggest you to keep an eye on https://biomejs.dev/internals/language-support/

**Comment by Netail** [comment]
Hey!
As of [Biome 2.3](https://biomejs.dev/blog/biome-v2-3/) we support HTML, Astro, Svelte & Vue under the experimental option: `html.experimentalFullSupportEnabled`.
Support is still experimental due to potentially missed edge cases, but definitely worth trying :) 
In case you find any issues please make sure to [create a GitHub issue](https://github.com/biomejs/biome/issues/new/choose) so we can get closer to stable :)

More on HTML-ish languages can be found here: https://biomejs.dev/internals/language-support/#html-super-languages-support
